web/static/js/components/stage_progression_button.jsx

```diff
@@ -33,11 +36,15 @@ export class StageProgressionButton extends Component {
       buttonDisabled,
       className,
       currentUser,
+      presences,
       retroUpdateRequested,
       config,
+      votes,
     } = this.props
 
-    const { modalOpen, showSubmitIdeaPrompt } = this.state
+    const { modalOpen } = this.state
+    const showAllVotesInPrompt = !isEmpty(presences)
+      && (votes.length >= VOTE_LIMIT * presences.length)
```

Comment:
We calculate the total number of maximum possible votes across all users ( `presences.length` * `VOTE_LIMIT`) as well as the current number of votes cast so far (`votes.length`). If they are equal we know to display the tooltip.